tests/rados/test_config_parameter_chk.py

```diff
@@ -0,0 +1,122 @@
+"""
+This method contains the scenarios to the the Mclock
+"""
+from configparser import ConfigParser
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+
+log = Log(__name__)
+ini_file = "conf/quincy/rados/test-confs/rados_config_parameters.ini"
+
+
+def run(ceph_cluster, **kw):
+    """
+    Automates the following test scenarios.
+    1. Checking the MSGR V2 in 5.x
+    2. Checking the MSGR V2 in 6.x
+    3. Verifying the all *sleep* parameters are set to 0
+    4. In Mclock the  reservation, weight and limit parameters are not modifiable
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    osd_list = []
+    mclock_profile = ["balanced", "high_recovery_ops", "high_client_ops"]
+    # To get the parameters from the ini
+    rados_object = RadosOrchestrator(node=cephadm)
+    mon_object = MonConfigMethods(rados_obj=rados_object)
+
+    config_info = ConfigParser()
+    config_info.read(ini_file)
+
+    # To get the OSD list
+    ceph_nodes = kw.get("ceph_nodes")
+    for node in ceph_nodes:
+        if node.role == "osd":
+            node_osds = rados_object.collect_osd_daemon_ids(node)
+            osd_list = osd_list + node_osds
+    if config.get("scenario") == "msgrv2_5x":
+        msgr_parameter = config_info.items("MSGRV2")
+        result = config_param_checker(mon_object, osd_list, msgr_parameter)
+        if result:
+            log.error("The MSGRV2 parameters are exist in the 5.x build")
+            return 1
+        return 0
+    if config.get("scenario") == "msgrv2_6x":
+        msgr_parameter = config_info.items("MSGRV2")
+        result = config_param_checker(mon_object, osd_list, msgr_parameter)
+        if not result:
+            log.error("The MSGRV2 parameter value is not correct in the 6.x build")
+            return 1
+        return 0
+
+    # Checking all sleep parameters in mclock profiles
+    if config.get("scenario") == "mclock_sleep":
```

Comment:
Test case speaks about trying to modify the values of *sleep* values. WE are just checking the default Values.

Update needed in automation/ Test case?

Reply:
Referring to test CEPH-83574903 here.

Reply:
updated the automation script.